src/monika/Shop.java

```diff
@@ -0,0 +1,106 @@
+package monika;
+class Shop{
+
+   int maggieStock=50;
+   int dosaStock=43;
+   int pouchesStock=30;
+   int panipuriStock=43;
+   int masalaStock=74;
+   
+   
+   
+   void customerBoughtItems(int maggieCount, int dosaCount, int pouchesCount ,int panipuriCount, int masalaCount){
+       if (maggieStock !=0 && maggieCount<= maggieStock)
+	       maggieStock=maggieStock-maggieCount;
+		   else if (maggieStock<=maggieCount)
```

Comment:
If you check your code then second condition in if statement & else if() both are same, it neven goes into  **else if()** condition. same for all other condition.